src/components/custom/dashboard/client/pages/diet-tracker/MealDetailModal.jsx

```
@@ -2,344 +2,381 @@
+            </div>
+
+            {/* Quick Select Buttons */}
+            <div className="flex items-center gap-3 flex-wrap justify-center">
+              <button
+                className={`px-4 py-2 rounded-lg font-medium ${
+                  portion === 0.5
+                    ? "bg-gradient-to-r from-[#FF6B00] to-[#FF9A00] text-white"
+                    : "bg-gradient-to-br from-zinc-800 to-zinc-900 border border-zinc-700/50 text-white hover:bg-zinc-700"
+                }`}
+                onClick={() => setPortion(0.5)}
+              >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)